src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -998,25 +999,26 @@ export class ProgrammingExerciseUpdateComponent implements OnInit {
         this.selectedProjectType = history.state.programmingExerciseForImportFromFile.projectType;
     }
 
-    getInfoStepInputs() {
+    getProgrammingExerciseCreationConfig(): ProgrammingExerciseCreationConfig {
         return {
+            isImportFromFile: this.isImportFromFile,
```

Comment:
same as above, can you please group the import related parameters closer together